.azure-pipelines/linux-conda-CI.yml

```diff
@@ -44,11 +44,17 @@ jobs:
       git clone --recursive https://github.com/cjlin1/libsvm libsvm
       cd libsvm
       make lib
+      mkdir dependencies && cd dependencies
+      git clone https://github.com/vinitra/onnxmltools
+      cd onnxmltools/onnxutils
+      git fetch && git checkout refactor-common
+      pip install -e .
     displayName: 'Install dependencies'
```

Comment:
I've currently been installing onnxtk from the source of this PR in my fork as the dependencies have shifted. Once this has been checked into master, then it makes sense to install onnxtk from the source of the onnx/onnxmltools repo as well as publish a new version of the [onntk package to pypi](https://pypi.org/project/onnxtk/), ideally with more cohesive documentation.

Reply:
the onnxmltools local clone should already have any change in this PR? can't we just cd onnxmltools/onnxutils to install it? why do we have to install it from your fork?

Reply:
Ah, that makes a lot more sense! I didn't realize that the CI build creates a local clone as I didn't see it expressed in the yml files. Updated with a fix.